x-pack/plugins/synthetics/public/apps/synthetics/components/settings/global_params/params_list.tsx

```diff
@@ -0,0 +1,273 @@
+          // TODO: add link to docs
+          defaultMessage="Define variables and parameters that you can use in the configuration of browser and lightweight monitors, such as credentials or URLs. {learnMore}"
+          values={{
+            learnMore: (
+              <EuiLink
+                href="https://www.elastic.co/guide/en/observability/master/synthetics-params-secrets.html"
+                target="_blank"
+              >
+                {LEARN_MORE}
+              </EuiLink>
+            ),
+          }}
+        />
+      </EuiText>
+      <EuiSpacer size="m" />
+      <EuiInMemoryTable<SavedObject<SyntheticsParam>>
+        itemId={'id'}
+        loading={loading}
+        tableCaption={PARAMS_TABLE}
+        items={items}
+        columns={columns}
+        tableLayout="auto"
+        isSelectable={true}
+        pagination={true}
+        sorting={true}
+        childrenBetween={
+          <TableTitle
+            pageIndex={pageIndex}
+            pageSize={pageSize}
+            total={items.length}
+            label={PARAMS_LABEL}
+          />
+        }
+        onTableChange={({ page }) => {
+          setPageIndex(page?.index ?? 0);
+          setPageSize(page?.size ?? 10);
+        }}
+        selection={{
+          selectable: () => true,
+          onSelectionChange: (sItems) => {
+            setSelectedItems(sItems);
+          },
+          initialSelected: selectedItems,
+        }}
+        search={{
+          toolsLeft: renderToolsLeft(),
+          toolsRight: renderToolRight(),
+          box: {
+            incremental: true,
+          },
+          filters: [
+            {
+              type: 'field_value_selection',
+              field: 'attributes.tags',
+              name: 'Tags',
+              multiSelect: true,
+              options: [...tagsList].map((tag) => ({
+                value: tag,
+                name: tag,
+                view: tag,
+              })),
+            },
+          ],
+        }}
+        message={loading ? LOADING_TEXT : undefined}
+      />
+      {isDeleteModalVisible && deleteParam && (
+        <DeleteParam
+          items={deleteParam}
+          setIsDeleteModalVisible={setIsDeleteModalVisible}
+          setRefreshList={setRefreshList}
+        />
+      )}
+    </div>
+  );
+};
+
+const PARAMS_TABLE = i18n.translate('xpack.synthetics.settingsRoute.params.tableCaption', {
+  defaultMessage: 'Synthetics Global Parameters',
+});
+
+const PARAMS_LABEL = i18n.translate('xpack.synthetics.settingsRoute.params.label', {
+  defaultMessage: 'Parameters',
+});
+
+const LEARN_MORE = i18n.translate('xpack.synthetics.settingsRoute.params.learnMore', {
+  defaultMessage: 'Learn more.',
+});
+
+const EDIT_PARAM = i18n.translate('xpack.synthetics.settingsRoute.params.editLabel', {
+  defaultMessage: 'Edit Parameter',
+});
+
+const LOADING_TEXT = i18n.translate('xpack.synthetics.settingsRoute.params.loading', {
+  defaultMessage: 'Loading ...',
```

Comment:
```suggestion
  defaultMessage: 'Loading...',
```
I think ellipses should be attached the word. Not sure though.